packages/xo-server-web-hooks/src/index.js

```diff
@@ -163,6 +213,13 @@ export const testSchema = {
       type: 'string',
       description: 'The URL the test request will be sent to',
     },
+    format: {
+      default: 'json',
+      description: 'The format of the payload sent to the URL',
+      enum: ['json', 'office365'],
+      enumNames: ['JSON', 'Office 365 connector'],
+      title: 'Payload format',
+    },
```

Comment:
IMHO, the test should use the format from the configuration. The main goal of a test is to make sure the plugin was configured properly, so it should simulate a real world run as much as possible.